edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/EdgeOperator.cs

```diff
@@ -327,7 +341,7 @@ Option<V1Service> GetServiceFromModule(Dictionary<string, string> labels, Kubern
             if (portList.Count > 0)
             {
                 // Selector: by module name and device name, also how we will label this puppy.
-                var objectMeta = new V1ObjectMeta(labels: labels, name: KubeUtils.SanitizeDNSValue(module.ModuleIdentity.ModuleId));
+                var objectMeta = new V1ObjectMeta(annotations: serviceAnnotations.GetOrElse(() => null), labels: labels, name: KubeUtils.SanitizeDNSValue(module.ModuleIdentity.ModuleId));
```

Comment:
Nit: You maybe able to say `serviceAnnotations.OrDefault()` instead of `serviceAnnotations.GetOrElse(() => null)`.

Reply:
You are correct, I think this is a general cleanup, because I used GetOrElse all over this file.